### PR TITLE
Image sharing react issue1

### PR DESCRIPTION
**Pull request by laeschjs**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: app/javascript/image_sharing/components/App.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 class App extends Component {
   /* Add Prop Types check*/
   render() {
     return (
       <div>
-        <Header title={'Tell us what you think'} />
-        /* Put your components here: Flash Message, Form, Footer */
+        <Header title='Tell us what you think' />
+        {/* Put your components here: Flash Message, Form, Footer */}
+        <Footer content='Copyright &copy;: Appfolio Inc. Onboarding' />
```
Comment (bhargav265) [inline app/javascript/image_sharing/components/App.js]: We need to add a test to `App.spec.js` confirming that it renders Footer

File: app/javascript/image_sharing/test/components/Footer.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+import Footer from '../../components/Footer';
+
+describe('<Footer />', () => {
+  it('renders Footer text', () => {
+    const text = 'test footer text';
+    const wrapper = shallow(<Footer content={text} />);
+    expect(wrapper.find('#js-footer-text').text()).to.contain(text);
```
Comment (bhargav265) [inline app/javascript/image_sharing/test/components/Footer.spec.js]: I think we can do to.equal to ensure that it renders just that text

Comment (laeschjs) [reply app/javascript/image_sharing/test/components/Footer.spec.js]: fixed

File: app/javascript/image_sharing/components/App.js (hunk 1 above)
Comment (laeschjs) [reply app/javascript/image_sharing/components/App.js]: fixed

File: app/javascript/image_sharing/test/components/App.spec.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import App from '../../components/App';
+import Header from '../../components/Header';
+import Footer from '../../components/Footer';
+
+describe('<App />', () => {
+  it('renders Header and Footer', () => {
+    const app = shallow(<App stores={null} />).dive();
+    const header = <Header title='Tell us what you think' />;
+    const footer = <Footer content='Copyright &copy;: Appfolio Inc. Onboarding' />;
+    expect(app.contains(header)).to.equal(true);
```
Comment (bhargav265) [inline app/javascript/image_sharing/test/components/App.spec.js]: This works, but a better way to test is to find the `Header` component in the app component. and the assert the props of the Header component

Comment (bhargav265) [reply app/javascript/image_sharing/test/components/App.spec.js]: Same for the footer

File: package.json [hunk 4]
```diff
@@ -10,7 +10,8 @@
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "react-scripts": "^3.2.0",
-    "reactstrap": "^7.1.0"
+    "reactstrap": "^7.1.0",
+    "whatwg-fetch": "^3.0.0"
```
Comment (bhargav265) [inline package.json]: probable merge this in the first commit

Comment (laeschjs) [reply package.json]: fixed

File: app/javascript/image_sharing/test/components/App.spec.js (hunk 3 above)
Comment (laeschjs) [reply app/javascript/image_sharing/test/components/App.spec.js]: fixed

**Pull request merged**